server/src/templates/agent_detail.html

```diff
@@ -2,17 +2,27 @@
 {% set active_page = 'agents' %}
 {% block content %}
 <div class="center">
+    <p>
     Agent Name: {{ agent.name }}<br>
     State: {{ agent.state }}<br>
     Location: {{ agent.location }}<br>
     Last Updated: {{ agent.updated_at.strftime('%Y-%m-%d %H:%M:%S') }}<br>
-    <br>
-    <br>
+    </p>
+    <p>
+    Queues:
+    <ul>
+        {% for queue in agent.queues %}
+        <li><a href="/queues/{{ queue }}">{{ queue }}</a></li>
+        {% endfor %}
+    </ul>
+    </p>
```

Comment:
I'm not aware what are advertised queues and how they are different from rest of queues. But would it be useful to show them separately or mark the queues that are advertised?

Reply:
Not really - in this case, we probably always should have been using all queues rather than just the advertised ones. Also, I expect advertised queues will work very differently in the future.
What this will allow is looking at an agent page and seeing all the possible queues you can use to submit a job there, which I think will be quite useful for end-users!